static/app/components/events/autofix/autofixChanges.tsx

```diff
@@ -36,7 +37,9 @@ function AutofixRepoChange({
   const api = useApi();
   const queryClient = useQueryClient();
 
-  const {mutate: createPr, isLoading} = useMutation({
+  const [hasClickedCreatePr, setHasClickedCreatePr] = useState(false);
+
+  const {mutate: createPr} = useMutation({
```

Comment:
i think useMutation has a state you can use instead of your own? It should have isPending or something which i think would behave the same as yours

Reply:
Hmm I only see `isLoading`, which is what this replaces. Because the autofix state comes from a polling query in a far parent we're basically waiting for the pr state to be present here before the loader goes away.